### PR TITLE
[feat] members section

### PR DESCRIPTION
**Pull request by doyourbest96**

### CODE REVIEW

**Review by jamest33 — COMMENTED** [review]
Everything is ok.

**Review by jamest33 — COMMENTED** [review]
Next step is footer.

File: src/components/MemberCard/MemberCard.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import Image from "next/image";
+
+interface MemberCardProps {
+  avatar: string;
+  name: string;
+  title?: string;
+  note?: string;
+}
```
Comment (BlueOcean5911) [inline src/components/MemberCard/MemberCard.tsx:8]: It should be saved in types folder. Fucking!

**Review by BlueOcean5911 — COMMENTED** [review]
Cannot Merge!!! Remove Fucking label!

**Review by BlueOcean5911 — COMMENTED** [review]
good

**Pull request merged**